lib/tasks/dev.rake

```diff
@@ -0,0 +1,39 @@
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate developers test data'
+  task generate_data: :environment do
+
+    100.times do
+      email = Faker::Internet.email
+      Developer.find_or_create_by(email: email)
```

Comment:
At here, we have `FactoryBot`
I think you should do create some `spec/factories`, it will help you create dummy data easy

https://github.com/thoughtbot/factory_bot_rails

Reply:
Thanks @thienhv 
I have add FactoryBot for developers and use it to generate data in rake task
Can you help me reviewing it again ?